kairos-contracts/demo-contract/contract/src/utils/events.rs

```diff
@@ -0,0 +1,11 @@
+use casper_event_standard::Event;
+extern crate alloc;
+use alloc::string::String;
+use casper_types::Key;
+
+#[derive(Event)]
+pub struct Deposit {
+    pub depositor: Key,
+    pub amount: u64,
+    pub timestamp: Option<String>,
```

Comment:
@Avi-D-coder The only last thing I'm concerned about is this. We set it to `None` right now, and `String` is not typesafe

Reply:
Do we even want a Timestamp? We might want to get rid of it entirely?

Reply:
i don't need it, i agree with removing it

Reply:
removed [b663b89](https://github.com/cspr-rad/kairos/pull/95/commits/b663b89cb45a04c4a27633ea7859b5a646f7c51a)